Mathlib/Algebra/Algebra/Basic.lean

```diff
@@ -431,7 +432,7 @@ theorem coe_linearMap : ⇑(Algebra.linearMap R A) = algebraMap R A :=
   rfl
 #align algebra.coe_linear_map Algebra.coe_linearMap
 
-instance id : Algebra R R :=
+@[reducible] instance id : Algebra R R :=
```

Comment:
I thought the whole point in `instance` is that it's reducible enough for anything typeclass related?

Reply:
Apparently not

Reply:
Do you have a mwe to demo that?

(I agree with the principle behind the rest of this PR to change the non-instances to be reducible)

Reply:
Ah processing what you are saying now. You are correct, that is silly.